### PR TITLE
update several packages to unicode 15.0.0

### PR DESCRIPTION
**Pull request by kilianar**
###### Description of changes
[Unicode 15](http://blog.unicode.org/2022/09/announcing-unicode-standard-version-150.html) was just released. This pr bumps all Unicode dependencies that i found from 14 to 15.

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by kilianar** [comment]
Result of `nixpkgs-review pr 191246` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages marked as broken and skipped:</summary>
  <ul>
    <li>kdeplasma-addons</li>
    <li>libsForQt512.kdeplasma-addons</li>
    <li>libsForQt512.plasma-desktop</li>
    <li>libsForQt514.kdeplasma-addons</li>
    <li>libsForQt514.plasma-desktop</li>
    <li>plasma-desktop</li>
  </ul>
</details>
<details>
  <summary>113 packages built:</summary>
  <ul>
    <li>acgtk</li>
    <li>adapta-gtk-theme</li>
    <li>beluga</li>
    <li>bluefish</li>
    <li>chrome-gnome-shell</li>
    <li>emacsPackages.ac-mozc</li>
    <li>emacsPackages.mozc</li>
    <li>emacsPackages.mozc-cand-posframe</li>
    <li>emacsPackages.mozc-im</li>
    <li>emacsPackages.mozc-popup</li>
    <li>emacsPackages.mozc-temp</li>
    <li>enlightenment.econnman</li>
    <li>enlightenment.ecrire</li>
    <li>enlightenment.efl</li>
    <li>enlightenment.enlightenment</li>
    <li>enlightenment.ephoto</li>
    <li>enlightenment.evisum</li>
    <li>enlightenment.rage</li>
    <li>enlightenment.terminology</li>
    <li>flow</li>
    <li>fstar</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gucharmap</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>grub2 (grub2_light)</li>
    <li>grub2_efi</li>
    <li>grub2_pvgrub_image</li>
    <li>grub2_xen</li>
    <li>haxe</li>
    <li>haxePackages.format</li>
    <li>haxePackages.heaps</li>
    <li>haxePackages.hlopenal</li>
    <li>haxePackages.hlsdl</li>
    <li>haxePackages.hxcpp</li>
    <li>haxePackages.hxcs</li>
    <li>haxePackages.hxjava</li>
    <li>haxePackages.hxnodejs_4</li>
    <li>haxe_4_0</li>
    <li>haxe_4_1</li>
    <li>ibus</li>
    <li>ibus-engines.anthy</li>
    <li>ibus-engines.hangul</li>
    <li>ibus-engines.kkc</li>
    <li>ibus-engines.libpinyin</li>
    <li>ibus-engines.libthai</li>
    <li>ibus-engines.m17n</li>
    <li>ibus-engines.mozc</li>
    <li>ibus-engines.rime</li>
    <li>ibus-engines.table</li>
    <li>ibus-engines.table-chinese</li>
    <li>ibus-engines.table-others</li>
    <li>ibus-engines.typing-booster</li>
    <li>ibus-engines.typing-booster-unwrapped</li>
    <li>ibus-engines.uniemoji</li>
    <li>ibus-qt</li>
    <li>ibus-with-plugins</li>
    <li>libbap</li>
    <li>libsForQt5.kdeplasma-addons</li>
    <li>libsForQt5.plasma-desktop</li>
    <li>libsForQt5_openssl_1_1.kdeplasma-addons</li>
    <li>libsForQt5_openssl_1_1.plasma-desktop</li>
    <li>liquidsoap</li>
    <li>mate.mate-applets</li>
    <li>mate.mate-tweak</li>
    <li>mlterm</li>
    <li>mojave-gtk-theme</li>
    <li>ocamlPackages.bap</li>
    <li>ocamlPackages.bonsai</li>
    <li>ocamlPackages.higlo</li>
    <li>ocamlPackages.iri</li>
    <li>ocamlPackages.lambdapi</li>
    <li>ocamlPackages.piqi</li>
    <li>ocamlPackages.piqi-ocaml</li>
    <li>ocamlPackages.ppx_css</li>
    <li>ocamlPackages.sedlex</li>
    <li>ocamlPackages.xtmpl</li>
    <li>ocamlPackages.xtmpl_ppx</li>
    <li>openbangla-keyboard</li>
    <li>os-prober</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-keyboard</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>phosh</li>
    <li>polychromatic</li>
    <li>python310Packages.bap</li>
    <li>python310Packages.pythonefl</li>
    <li>python39Packages.bap</li>
    <li>python39Packages.pythonefl</li>
    <li>snippetpixie</li>
    <li>stog</li>
    <li>timeshift</li>
    <li>unicode-character-database</li>
    <li>unicode-emoji</li>
    <li>unicode-paracode</li>
    <li>unifont</li>
    <li>unifont_upper</li>
    <li>unihan-database</li>
    <li>vimix-gtk-themes</li>
    <li>whitesur-gtk-theme</li>
    <li>woeusb</li>
    <li>woeusb-ng</li>
  </ul>
</details>

**Comment by tjni** [comment]
Result of `nixpkgs-review pr 191246` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>bluefish</li>
    <li>liquidsoap</li>
  </ul>
</details>
<details>
  <summary>11 packages failed to build:</summary>
  <ul>
    <li>haxe (haxe_4_2)</li>
    <li>haxePackages.format</li>
    <li>haxePackages.heaps</li>
    <li>haxePackages.hlopenal</li>
    <li>haxePackages.hlsdl</li>
    <li>hxcpp (haxePackages.hxcpp)</li>
    <li>haxePackages.hxcs</li>
    <li>haxePackages.hxjava</li>
    <li>haxePackages.hxnodejs_4</li>
    <li>haxe_4_0</li>
    <li>haxe_4_1</li>
  </ul>
</details>
<details>
  <summary>14 packages built:</summary>
  <ul>
    <li>acgtk</li>
    <li>beluga</li>
    <li>flow</li>
    <li>fstar</li>
    <li>libbap</li>
    <li>python310Packages.bap</li>
    <li>python39Packages.bap</li>
    <li>stog</li>
    <li>unicode-character-database</li>
    <li>unicode-emoji</li>
    <li>unicode-paracode</li>
    <li>unifont</li>
    <li>unifont_upper</li>
    <li>unihan-database</li>
  </ul>
</details>

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 191246` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.kdeplasma-addons</li>
    <li>libsForQt512.plasma-desktop</li>
    <li>libsForQt514.kdeplasma-addons</li>
    <li>libsForQt514.plasma-desktop</li>
  </ul>
</details>
<details>
  <summary>105 packages built:</summary>
  <ul>
    <li>acgtk</li>
    <li>adapta-gtk-theme</li>
    <li>beluga</li>
    <li>bluefish</li>
    <li>chrome-gnome-shell</li>
    <li>emacsPackages.ac-mozc</li>
    <li>emacsPackages.mozc</li>
    <li>emacsPackages.mozc-cand-posframe</li>
    <li>emacsPackages.mozc-im</li>
    <li>emacsPackages.mozc-popup</li>
    <li>emacsPackages.mozc-temp</li>
    <li>enlightenment.econnman</li>
    <li>enlightenment.ecrire</li>
    <li>enlightenment.efl</li>
    <li>enlightenment.enlightenment</li>
    <li>enlightenment.ephoto</li>
    <li>enlightenment.evisum</li>
    <li>enlightenment.rage</li>
    <li>enlightenment.terminology</li>
    <li>flow</li>
    <li>fstar</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gucharmap (gnome.gucharmap)</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>grub2 (grub2_full ,grub2_light)</li>
    <li>grub2_efi</li>
    <li>grub2_pvgrub_image</li>
    <li>grub2_xen</li>
    <li>haskellPackages.gi-ibus</li>
    <li>haxe (haxe_4_2)</li>
    <li>haxePackages.format</li>
    <li>haxePackages.heaps</li>
    <li>haxePackages.hlopenal</li>
    <li>haxePackages.hlsdl</li>
    <li>hxcpp (haxePackages.hxcpp)</li>
    <li>haxePackages.hxcs</li>
    <li>haxePackages.hxjava</li>
    <li>haxePackages.hxnodejs_4</li>
    <li>haxe_4_0</li>
    <li>haxe_4_1</li>
    <li>ibus</li>
    <li>ibus-engines.anthy</li>
    <li>ibus-engines.hangul</li>
    <li>ibus-engines.kkc</li>
    <li>ibus-engines.libpinyin</li>
    <li>ibus-engines.libthai</li>
    <li>ibus-engines.m17n</li>
    <li>ibus-engines.mozc</li>
    <li>ibus-engines.rime</li>
    <li>ibus-engines.table</li>
    <li>ibus-engines.table-chinese</li>
    <li>ibus-engines.table-others</li>
    <li>ibus-engines.typing-booster</li>
    <li>ibus-engines.typing-booster-unwrapped</li>
    <li>ibus-engines.uniemoji</li>
    <li>ibus-qt</li>
    <li>ibus-with-plugins</li>
    <li>libbap</li>
    <li>libsForQt5.kdeplasma-addons (libsForQt515.kdeplasma-addons ,plasma5Packages.kdeplasma-addons)</li>
    <li>libsForQt5.plasma-desktop (libsForQt515.plasma-desktop ,plasma5Packages.plasma-desktop)</li>
    <li>libsForQt5_openssl_1_1.kdeplasma-addons</li>
    <li>libsForQt5_openssl_1_1.plasma-desktop</li>
    <li>liquidsoap</li>
    <li>mate.mate-applets</li>
    <li>mate.mate-tweak</li>
    <li>mlterm</li>
    <li>mojave-gtk-theme</li>
    <li>openbangla-keyboard</li>
    <li>os-prober</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-keyboard</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>phosh</li>
    <li>polychromatic</li>
    <li>python310Packages.bap</li>
    <li>python310Packages.pythonefl</li>
    <li>python39Packages.bap</li>
    <li>python39Packages.pythonefl</li>
    <li>snippetpixie</li>
    <li>stog</li>
    <li>tests.testers.nixosTest-example</li>
    <li>tests.trivial-builders.references</li>
    <li>timeshift</li>
    <li>unicode-character-database</li>
    <li>unicode-emoji</li>
    <li>unicode-paracode</li>
    <li>unifont</li>
    <li>unifont_upper</li>
    <li>unihan-database</li>
    <li>vimix-gtk-themes</li>
    <li>whitesur-gtk-theme</li>
    <li>woeusb</li>
    <li>woeusb-ng</li>
  </ul>
</details>

**Review by superherointj — APPROVED** [review]

File: pkgs/tools/misc/fntsample/default.nix [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 let
   ucd-blocks = fetchurl {
-    url = "https://www.unicode.org/Public/14.0.0/ucd/Blocks.txt";
+    url = "https://www.unicode.org/Public/15.0.0/ucd/Blocks.txt";
     hash = "sha256-WYhw3d73s0talykWUoxFav8nZbec1Plkf7WM63Z+fxc=";
```
Comment (jtojnar) [inline pkgs/tools/misc/fntsample/default.nix:21]: Looks like this has was not updated, the files are not the same.

**Pull request merged**